cmd/stack_yaml_lint.go

```diff
@@ -127,15 +127,15 @@ func (stackDetails *StackYaml) checkTemplatingData(log *LoggingConfig) (int, int
 	return stackLintErrorCount, stackLintWarningCount
 }
 
-func (stackDetails *StackYaml) checkLicense(log *LoggingConfig) int {
+func (stackDetails *StackYaml) checkLicense(config *RootCommandConfig) int {
```

Comment:
By only using `log *LoggingConfig` in the functions below, you can remove this change